api/api.proto

```diff
@@ -1379,13 +1381,14 @@ message SdkSchedulePolicyInterval {
   // Number of instances to retain
   int64 retain = 1;
 
+  // Start oneof at field number 200 to allow for expansion
   oneof period_type {
     // Daily policy
-    SdkSchedulePolicyIntervalDaily daily = 2;
+    SdkSchedulePolicyIntervalDaily daily = 200;
```

Comment:
how are these related to cred name change?

Reply:
Due to this change, I wanted to prepare the message for future ones that may come. I'm making sure we have enough field numbers to allow for expansion while trying to keep the oneof's at the end of the message. That way we can add more in the oneof's.

Long story short, this change brought up that we needed to make sure that our messages were easily enhanced. All others messages are ok, just the ones with oneof needed to be adjusted.